src/main/java/net/sf/jabref/openoffice/CitationManager.java

```diff
@@ -140,20 +140,20 @@ public void showDialog() {
     static class CitEntry implements Comparable<CitEntry> {
 
         private final String refMarkName;
-        private String pageInfo;
+        private Optional<String> pageInfo;
         private final String context;
-        private final String origPageInfo;
+        private final Optional<String> origPageInfo;
 
 
-        public CitEntry(String refMarkName, String context, String pageInfo) {
+        public CitEntry(String refMarkName, String context, Optional<String> optional) {
```

Comment:
I wouldn't have changed it unless I changed the method generating the argument. If I get you correct there should be two different constructors? One with Optional (for convenience) and one for String (for actual use)?